src/ucs/datastruct/bitmap.h

```diff
@@ -45,6 +45,17 @@ typedef uint64_t ucs_bitmap_word_t;
 #define _UCS_BITMAP_NUM_WORDS(_bitmap) ucs_static_array_size((_bitmap).bits)
 
 
+/**
+ * Get the number of bits in a given bitmap
+ *
+ * @param _bitmap Bits number in this bitmap
```

Comment:
it is just a bitmap itself, not a number of bits

Reply:
reworded